File: test/fixtures/python/redirect_variable_and_string_data.py
import os
import requests

headers = {
    'Content-Type': 'application/x-www-form-urlencoded',
}

data = 'foo'
with open('start' + os.getenv('FILENAME', '') + 'end') as f:
    data += '&' + f.read().replace('\n', '').replace('\r', '')

response = requests.post('http://localhost:28139', headers=headers, data=data.encode())

File: test/fixtures/r/redirect_variable_and_string_data.r
library(httr)

headers = c(
  `Content-Type` = "application/x-www-form-urlencoded"
)

data = paste0("foo&@start", Sys.getenv("FILENAME"), "$end")

res <- httr::POST(url = "http://localhost:28139", httr::add_headers(.headers=headers), body = data)
